src/firefly/js/ui/PopupUtil.jsx

```diff
@@ -97,17 +97,11 @@ export function showPopup({ID, content, title='Options', modal = false}) {
  *
  * @param {string | object} content can be a string or a react component
  * @param {string} title
- * @param {boolean} [isCopyable] true or false, if undefined, then the default to true if it is a string
- * @param [wrapperStyle]
+ * @param [sx]
  * @return {object}
  */
-export function showInfoPopup(content, title='Information', isCopyable=undefined, wrapperStyle) {
-    if (isUndefined(isCopyable)) isCopyable = isString(content);
-    var results= (
-        <PopupPanel title={title} >
-            {makeContent(content, isCopyable, wrapperStyle)}
-        </PopupPanel>
-    );
+export function showInfoPopup(content, title='Information', sx) {
+    const results= ( <PopupPanel title={title} > {makeContent(content, sx)} </PopupPanel> );
```

Comment:
This is not how I would imagine `sx` to be used.  If it's to be applied to InfoPopup, then `sx` should be passed to `<PopupPanel sx={sx}>`.  To target a specific nested component in `makeContent`, have `makeContent` declared it.  See `makeContent` for details.

Reply:
done